templates/search_derived/search_clients_last_seen_v1/query.sql

```diff
@@ -1,14 +1,7 @@
 WITH
   _derived_search_cols AS (
     SELECT
-      CASE -- we only care about certain normalized engines
-          WHEN engine IS NULL THEN 'none'
-          WHEN STARTS_WITH(engine, 'google') THEN 'google'
-          WHEN STARTS_WITH(engine, 'ddg') OR STARTS_WITH(engine, 'duckduckgo') THEN 'ddg'
-          WHEN STARTS_WITH(engine, 'bing') THEN 'bing'
-          WHEN STARTS_WITH(engine, 'yandex') THEN 'yandex'
-          ELSE 'other'
-      END AS short_engine,
+      udf_strict_normalize_search_engine(engine) AS short_engine,
```

Comment:
We can get normalized directly from scd if we backfill it but this should do for now.